docs/src/modules/developing/pages/development-process-java.adoc

```diff
@@ -80,5 +80,5 @@ https://docs.kalix.io/services/running-locally.html[running them locally] before
 
 After testing locally, deploy your service to Kalix using the CLI or the Console. The following pages provide information about deployment:
 
-* https://docs.kalix.io/projects/index.html[Working with Kalix deployment projects]
+* https://docs.kalix.io/services/deploy-service.html[Managing Kalix services]
```

Comment:
The Javascript documentation had both pages. It links to the page to learn how to create a project and to the page on how to deploy. 

I would do the same here.

Reply:
https://docs.kalix.io/developing/development-process-js.html#_deploy_to_kalix

Reply:
The page talks only about services but yeah, I guess a project link can't hurt.  I think the links at the bottom of the Javascript page are more useful.  Will change to match.